DevEdu.API/DevEdu.DAL/Models/NotificationDto.cs

```diff
@@ -1,8 +1,18 @@
-﻿namespace DevEdu.DAL.Models
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DevEdu.DAL.Enums;
+
+
+namespace DevEdu.DAL.Models
 {
-   public class NotificationDto : BaseDto
+    public class NotificationDto : BaseDto
     {
-        public int UserId { get; set; }
         public string Text { get; set; }
+        public UserDto User { get; set; }
+        public DateTime Date { get; set; }
+        public Role Role { get; set; } 
```

Comment:
I think it will be useful to add an ability to create a notification for the particular group
so please add public GroupDto Group here
and "GroupId" column in DB table
and GroupId property in input model